ansible/roles/amazon-cloudwatch-agent/tasks/redhat.yml

```diff
@@ -72,6 +77,7 @@
         name: collectd
         state: started
         enabled: yes
+      notify: restart collectd
```

Comment:
I'd normally expect to see this notify under steps which are changing the collectd configuration files and require a restart.  I think this step can just start and enable the service

Reply:
I was thinking about if the role is re-run on the basis that the collectd config 'probably' changed then this needs restarting to pick up the new config. I've not found an auto-load option in collectd yet

Reply:
If you re-run, it won't trigger the notify as this step will be green/unchanged, i.e. the service is already started and enabled.  So you need to put the notify on steps which actually change the collectd config

Reply:
Ah, I was wondering about that - will change it elsewhere when I start getting the windows config agent working :+1